src/tactic/omega/find_scalars.lean

```diff
@@ -0,0 +1,62 @@
+/-
+Copyright (c) 2019 Seul Baek. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Seul Baek
+
+A tactic which performs Fourier–Motzkin elimination to find 
+a contradictory linear combination of input constraints.
```

Comment:
Is there any overlap between this procedure and the one used in `linarith`? If possible, it would be good to have a single common infrastructure, although maybe these procedures are too tied in to their respective environments for this to be feasible. @robertylewis @skbaek

Reply:
In principle they could probably be unified. But the term representations are different -- in particular, the one in `linarith` can be lighter weight and meta because there's no proof by reflection. And the actual FM elimination step is only a handful of lines. I think it's probably not worth it.

Reply:
Jeremy made similar remarks about sharing the same procedure. This would make a lot of sense if the elimination step was long and heavily optimized; but my feeling is that the tactics will hit other bottlenecks like memory consumption before the point where such optimizations matter.